bin/fetchl10nstrings.js

```diff
@@ -0,0 +1,341 @@
+#! /usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+/**
+ * Create the string bundles and then save them to disk.
+ */
+/*jshint node:true, browser:false, esnext:true*/
+"use strict";
+const async = require("marcosc-async");
```

Comment:
Aha! I see it is: https://github.com/mozilla/remote-newtab/blob/42b549bbfe7652f624cf2cb8fd526e0de1edea96/package.json#L39

This is the node version of our async library? https://github.com/mozilla/remote-newtab/blob/42b549bbfe7652f624cf2cb8fd526e0de1edea96/src/js/lib/async.js